pkg/cnab/provider/action.go

```diff
@@ -13,7 +13,7 @@ import (
 
 // Shared arguments for all CNAB actions
 type ActionArguments struct {
-	// Name of the instance.
+	// Name of the installation.
 	Claim string
```

Comment:
While we are here, would you also change this field `Claim` to `Installation`? See my previous comment for why it was called claim, but it represented the same thing as "instance" and should be changed in this case.